tests/foreman/api/test_hostgroup.py

```diff
@@ -109,7 +109,7 @@ def test_verify_bugzilla_1107708(self):
         # We have two environments (one created after publishing and one more
         # was created after promotion), so we need to select promoted one
         environments = entities.Environment().search(
-            query={'organization_id': org.id}
+            query={'search': 'organization_id={0}'.format(org.id)}
```

Comment:
@abalakh  @sghai  This is a very common mistake to make, it's kinda redundant to repeat the word "search" in this command. How about we change the Nailgun to when there is NO  "search" key in query nailgun solve it internally?